analytics/endpoint_data_objects.go

```diff
@@ -0,0 +1,54 @@
+package analytics
+
+import (
+	"github.com/mxmCherry/openrtb"
+)
+
+type RequestType string
+
+const (
+	COOKIE_SYNC RequestType = "/cookie_sync"
+	AUCTION     RequestType = "/openrtb2/auction"
+	SETUID      RequestType = "/set_uid"
+	AMP         RequestType = "/openrtb2/amp"
+)
+
+//Loggable object of a transaction at /openrtb2/auction endpoint
+type AuctionObject struct {
+	Type      RequestType
```

Comment:
That is because a module like FileLogger dumps all transactions to any endpoint into the same file - helps identify to what endpoint the request was made. Not sure if it could help for other modules, can get rid of it if not needed.

Reply:
Seems like that would be an implementation detail of the FileLogger, then? If it takes logic which other modules would use too, the code could be written in utils functions.